examples/minecraft.rs

```diff
@@ -152,9 +148,14 @@ fn rotate_camera(
 #[cfg(feature = "bevy_wgpu")]
 fn handle_zoom(
     keys: Res<ButtonInput<KeyCode>>,
-    mut viewing_glass: ResMut<ShocoVoxViewingGlass>,
     mut angles_query: Query<&mut DomePosition>,
+    svx_data: Option<ResMut<ShocoVoxRenderData>>,
 ) {
+    if let Some(ref svx_data) = svx_data {
+        if svx_data.is_changed() {
+            println!("changed!! --> {:?}", svx_data.node_children[0..2].to_vec());
```

Comment:
At some point we may want to add `tracing` or the like to our dependencies

Reply:
That's just some temporary debug display, but I agree on the tracing dependencies!